test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs

```diff
@@ -263,11 +263,34 @@ public void TesteModelWithIListMarkdownContentAttributeShouldSucceed()
 
         #endregion
 
+        #region HrefAttribute
+
+        [Fact]
+        public void TestSimpleModelWithHrefAttributeShouldSucceed()
+        {
+            var model = new SimpleModel
+            {
+                Href = "linkTarget"
```

Comment:
yes, like what Markup does.

---
In reply to: [111536280](https://github.com/dotnet/docfx/pull/1543#discussion_r111536280) [](ancestors = 111536280)